CONTRIBUTING.md

```diff
@@ -146,12 +146,17 @@ $ TEST_GREP=transformation make test
   <summary>More options</summary>
```

Comment:
Actually I am fine with removing the `details` tag, to make it more visible. Many, if not most, Babel fixture folders have `-` in their name.

Reply:
Yeah you right ^^